docs/reference/runtime/Memory.md

```diff
@@ -51,25 +51,25 @@ This is a basic overview of the memory model in Mbed OS.
 
 ```
 
-There are, at least, two kinds of memory in the system: flash and RAM.
+There are at least two kinds of memory in the system: flash and RAM.
 
 ### RAM
 
-Inside RAM we can distinguish two logical types: static and dynamic memory. Each of them is used in different ways:
-* Static (zero initialized)
-  * Vector table
-  * Global data
-  * Static data
-  * Stacks for default threads (main, timer, idle, scheduler/ISR)
-* Dynamic
-  * Heap (dynamic data)
+Inside RAM, we can distinguish two logical types: static and dynamic memory. You can use each of them in different ways:
```

Comment:
I think that 'you can use each of them' changes the meaning a bit. It's used by the system in this way rather than provides an option for the user.

Reply:
Ah. Good to know. Would "The system uses each of them in different ways:" be better?

Reply:
yeah, that sounds great.